### ISSUE TITLE
Request failed with status code 404

### ISSUE THREAD

**Issue by ghost** [AuroPick/epic-free-games#21]
After updating today's games, any search you run with any region the API returns this error, I was making a command yesterday and it was working, then today after updating this week's new games it gave me this

```
Erro: Error: An error occurred
         error: {
       "message": "CatalogOffer/offerMappings: Request failed with status code 404",
       "locations": [
               {}
       ],
       "correlationId": "1792c500-beb8-4854-9d69-43e4a9a30e54",
       "serviceResponse": "{\"errorMessage\":\"The item or resource being requested could not be found.\",\"errorCode\":\"errors.com.epicgames.not_found\",\"numericErrorCode\":1004,\"errorStatus\":404}",
       "stack": null,
       "path": [
               "Catalog",
               "searchStore",
               "elements",
               3,
               "offerMappings"
       ]
 },{
       "message": "CatalogNamespace/mappings: Request failed with status code 404",
       "locations": [
               {}
       ],
       "correlationId": "1792c500-beb8-4854-9d69-43e4a9a30e54",
       "serviceResponse": "{\"errorMessage\":\"The item or resource being requested could not be found.\",\"errorCode\":\"errors.com.epicgames.not_found\",\"numericErrorCode\":1004,\"errorStatus\":404}",
       "stack": null,
       "path": [
               "Catalog",
               "searchStore",
               "elements",
               3,
               "catalogNs",
               "mappings"
       ]
 }
```

**Comment by chrom007** [comment]
Same error... But request have valid data after error field 🤔

**Comment by AuroPick** [comment]
I have just released v4.0.2. Please Install the new version.
